pkg/ansible/watches/watches_test.go

```diff
@@ -637,80 +639,101 @@ func TestAnsibleVerbosity(t *testing.T) {
 	}
 }
 
```

Comment:
IHMO: We should add a test for the load [here](https://github.com/operator-framework/operator-sdk/blob/8522f1f716da086064d28a4e7af450e44c186da2/pkg/ansible/watches/watches_test.go#L100) with collections as well. However, if the @fabianvf think that it is not required too I am ok with.

Reply:
I'd say give it a try, if it's quick then :+1: but I don't think it blocks merge